app/components/settings/billing-page/status-pill.hbs

```diff
@@ -0,0 +1,19 @@
+<div
+  class="inline-flex items-center gap-2 rounded-full px-2 py-1 border mb-4
```

Comment:
```suggestion
  class="inline-flex items-center gap-2 rounded-full px-2 py-1 border
```

@Arpan-206 the `mb-4` is something we'd want to move out and pass in from the parent template instead. It can seem slightly duplicative to have `mb-4` everywhere this is used, but it follows the principle that anything affecting parent component layout is the parent component's responsibility